tutorials/hamiltonian_simulation/hamiltonian_simulation_guide.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<p>Nice plot. I would increase <code>num_qdrift</code> , or, alternatively, use <code>num_qdrift=30</code> as you did but run the simulation several times to get some statistics (this is eventually a random approach). You can use <code>ExecutionSession</code> .  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>